exonum-java-binding-core/rust/ejb-app/TUTORIAL.md

```diff
@@ -59,7 +59,7 @@ with a few additional parameters.
 #### Generate Template Config
 
 ```$sh
-$ ejb-app generate-template testnet/common.toml
+$ cargo run -- generate-template testnet/common.toml
```

Comment:
In which directory are these commands allowed? `ejb-app` is available in any, I'm not so sure about these.

Reply:
Then these instructions require a note of that (with a comment explaining that this is a temporary measure).